flujo/application/runner.py

```
@@ -511,6 +534,249 @@ async def replay_from_trace(self, run_id: str) -> PipelineResult[ContextT]:
+    ) -> list[PipelineResult[ContextT]]:
+        """Retry failed background tasks for a given parent run."""
+        results: list[PipelineResult[ContextT]] = []
+        failures: list[tuple[str, Exception]] = []
+        tasks = await self.get_failed_background_tasks(parent_run_id=parent_run_id)
+        for task in tasks:
+            for attempt in range(max_retries):
+                try:
+                    result = await self.resume_background_task(task.task_id)
+                    results.append(result)
+                    break
+                except Exception as exc:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)